### PR TITLE
Update group descriptions

### PR DESCRIPTION
**Pull request by qwertyone**

### CODE REVIEW

File: governance/groups.md [hunk 1]
```diff
@@ -7,32 +7,48 @@ A list of groups and members follows.
 ### Fix live bugs
 ### Committing changes to production server
 ## Quality review groups
-A list of groups and members follows.
+A list of groups and members follows. The wording of this text will be subject to revision as necessary as determined by the members of the teams upon introduction into the governance documents.
```
Comment (neilthemathguy) [inline governance/groups.md:10]: Again this needs more information and review from other members before it is been added to the governance process. Please follow the process, answer the respective questions.

#53

Thanks.

This comments applies to all the changes below.

Comment (qwertyone) [reply governance/groups.md:10]: It was. You were not there.

**Comment by neilthemathguy** [comment]
I'm very very sick of this attitude and immaturity you have shown in this comment. You know that I'm in Brussels attending the event and presenting Daemo. I could not attend the meetings. I didnt see you in many other meetings when decisions needed to make. Besides that I mention I will take a look as soon as I get time. Did you answer the questions that were raised? You didn't.

File: governance/groups.md (hunk 1 above)
Comment (neilthemathguy) [reply governance/groups.md:10]: I'm very very sick of this attitude and immaturity you have shown in this comment. You know that I'm in Brussels attending the event and presenting Daemo. I could not attend the meetings. I didnt see you in many other meetings when decisions needed to make. Besides that I mention I will take a look as soon as I get time. Did you answer the questions that were raised? You didn't.

**Pull request closed without merging**